tests/ui/DateEditor.test.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * @jest-environment jsdom
+ */
+import { type RenderResult, fireEvent, render } from '@testing-library/svelte';
+import moment from 'moment/moment';
+import { TASK_FORMATS } from '../../src/Config/Settings';
+import DateEditor from '../../src/ui/DateEditor.svelte';
+import DateEditorWrapper from './DateEditorWrapper.svelte';
+
+import { getAndCheckRenderedElement } from './RenderingTestHelpers';
+
+window.moment = moment;
+
+function renderDateEditor() {
```

Comment:
@claremacrae not sure if we need to keep this function and the `describe()` below. I think we don't need this. If you agree, feel free amend this PR and remove this code.

Reply:
Thanks for pointing that out - yes, we talked above remove it if the wrapper was successful - and then I forgot...
I will remove it.